HHS/Head-Start-TTADP
# [TTAHUB-934] Update approved report template

## Description of change
- Add a "version" column to the AR, which will determine which approved AR form to display
- Create new component to match [the mockup](https://www.figma.com/file/rJGMygyqMz8jjCcoEpzcnK/Activity-Reports?node-id=3786%3A123568) for the redesigned AR. (It's "Approved Activity Report - no borders" at the top)

## How to test
- Run the migration, confirm that all existing reports have either 0 or 1 as their version number, depending on whether they were legacy reports.
- Check existing reports, ensure that the layout on the approved report page hasn't changed. Also verify print view still looks ok.
- Create a new report, approve it, and view the approved report template. Match it to the mockup. Check the print view, make sure that it looks ok. You can verify that it has the correct version.

## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-934


## Checklists

### Every PR



### Production Deploy


### After merge/deploy
